.github/workflows/preDeploy.yml

```diff
@@ -208,26 +208,26 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
 
-  # Check if actor is member of Expensify organization by looking for expensify-expensify team
+  # Check if actor is member of Expensify organization by looking for Expensify/expensify team
   isExpensifyEmployee:
     runs-on: ubuntu-latest
 
     outputs:
-      isExpensifyEmployee: ${{ fromJSON(steps.checkActor.outputs.isTeamMember) }}
+      IS_EXPENSIFY_EMPLOYEE: ${{ fromJSON(steps.checkActor.outputs.isTeamMember) }}
 
     steps:
-      - name: Check whether the actor is member of expensify-expensify team
+      - name: Check whether the actor is member of Expensify/expensify team
         id: checkActor
         uses: tspascoal/get-user-teams-membership@baf2e6adf4c3b897bd65a7e3184305c165aec872
         with:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           username: ${{ github.actor }}
-          team: expensify-expensify
+          team: Expensify/expensify
 
   newContributorWelcomeMessage:
     runs-on: ubuntu-latest
     needs: isExpensifyEmployee
-    if: ${{ github.actor != 'OSBotify' && !fromJSON(needs.isExpensifyEmployee.outputs.isExpensifyEmployee) }}
+    if: ${{ github.actor != 'OSBotify' && !fromJSON(needs.isExpensifyEmployee.outputs.IS_EXPENSIFY_EMPLOYEE) }}
```

Comment:
Should this be an `OR` instead of an `AND`?

Reply:
`OR` would let `OSBotify` through no? OSBotify is not a member of the team so if it will be OR the other condition will always be true.